Detectors/TPC/reconstruction/src/KrBoxClusterFinder.cxx

```diff
@@ -13,15 +13,51 @@
 /// \author Philip Hauer <hauer@hiskp.uni-bonn.de>
 
 #include "TPCReconstruction/KrBoxClusterFinder.h"
+#include "TPCBase/CalDet.h"
+#include "TPCBase/Utils.h"
+
 #include "Framework/Logger.h"
 
+#include <TFile.h>
+#include <vector>
+
 using namespace o2::tpc;
 
 // Constructor:
-// This function creates a three dimensional vector (Pad,Row,Time) which is
-// filled with the charges
-KrBoxClusterFinder::KrBoxClusterFinder(std::vector<o2::tpc::Digit>& eventSector)
+// The constructor allocates a three dimensional array (Pad,Row,Time) which is
+// later filled with the recorded charges for each digit
+KrBoxClusterFinder::KrBoxClusterFinder()
 {
+}
```

Comment:
sorry, I overlooked this. The constructor now should be initialized with `= default` as I wrote in the header and removed here. This is I think why the CI fails.
```suggestion
```